### DISCUSSION TITLE
How to use cross-validation in Autokeras.

### DISCUSSION THREAD

**Discussion by wang-k-ui** [keras-team/autokeras#1434] [Q&A]
I only have a small sample of data (500, with 100 validation sets), and I want to evaluate the model using k-fold cross validation.

At present, the accuracy of training set and verification set is very different.

Who can tell me how to implement cross validation in Autokeras?

**Comment by haifeng-jin** [comment]
It is not supported in AutoKeras yet. It is in the plan of Keras Tuner. After it is supported in Keras Tuner, AutoKeras can supported directly.

**Reply by RaedMatti** [reply]
> It is not supported in AutoKeras yet. It is in the plan of Keras Tuner. After it is supported in Keras Tuner, AutoKeras can supported directly.

Hello Dear, is it supported now?!

**Comment by wang-k-ui** [comment]
您的邮件我已收到，会尽快处理！I have received your email and will deal with it as soon as possible!
